ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -103,7 +113,7 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
   const getGroupTenancyType = (group: MhrRegistrationHomeOwnerGroupIF): HomeTenancyTypes => {
     const numOfOwnersInGroup = group.owners.filter(owner => owner.action !== ActionTypes.REMOVED).length
 
-    if (numOfOwnersInGroup > 1) {
+    if (numOfOwnersInGroup > 1 && !hasETAGroupError(group)) {
```

Comment:
Same as above, designs call for Tenancy Type of NA in any scenario where there is a E T or A?